### PR TITLE
FIX add test combining onlyChangedAttrs true and alterationType/dateCreated/dateModified

### PR DESCRIPTION
**Pull request by fgalan**
This PR includes a .test showing the combined behaviour of onlyChangedAttrs true and alterationType notification.

At the present moment, if onlyChangedAttrs: true is used, alterationType is not included.

This PR can resolve the WIP in two ways:

1. Keeping the current behaviour, in which case:
   - Keep the .test as it is now
   - Document the behaviour in orion-api.md
2. Change CB behaviour, in which case
   - Change code to change the behaviour (including CNR entry)
   - Change the .test to implement the align with new behaviour
   - Document the behaviour in orion-api.md

Feedback welcome

### CODE REVIEW

**Comment by fgalan** [comment]
After internal discussion, let's go for option 2.

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - Add: log deprecation traces for usage of attrsFormat legacy in subscriptions
 - Add: deprecatedFeatures counters block in GET /v2/statistics
 - Fix: do not show dateExpires built-in attribute in GET /v2/types and GET /v2/types/{type} operations (#4451)
+- Fix: builtin attributes alterationType, dateCreated and dateModified included in notifications even when onlyChangedAttrs was true
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:8]:
```suggestion
- Fix: builtin attributes alterationType, dateCreated and dateModified included in notifications even when onlyChangedAttrs is true
```

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR does some modification to English documentation. It would be great if you could sync Japanese translation, please. Thanks!

(Maybe you could unify changes https://github.com/telefonicaid/fiware-orion/pull/4477, https://github.com/telefonicaid/fiware-orion/pull/4476, etc. into one PR. Or do one-by-one PR. As you prefer :)

**Comment by fisuda** [comment]
I sent the PR #4484. Thanks.

**Pull request merged**